### PR TITLE
new: updating release system and adding labels

### PR DESCRIPTION
**Pull request by jriddle-linode**
## 📝 Description

**What does this PR do and why is this change necessary?**

Adds a label and release configuration file, adding labels to PRs will now correctly organize them in the Github Release notes.

### CODE REVIEW

**Comment by jriddle-linode** [comment]
The label i should use for this doesn't exist by default and wants me to create a new one until this is merged into main so ironically i won't be able to use it on this release..

File: .github/release.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+changelog:
+  exclude:
+    labels:
+      - ignore-for-release
```
Comment (lgarber-akamai) [inline .github/release.yml:4]: Nice! Could `ignore-for-release` also be added to labels.yml?

**Review by lgarber-akamai — APPROVED** [review]
LGTM other than the requested changes above, nice work!

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  color: d3e1e6
+- name: testing
+  description: for updates to the testing suite in the changelog.
+  color: 933ac9
+- name: breaking-change
+  description: for breaking changes in the changelog.
+  color: ff0000
+- name: ignore-for-release
+  description: PRs you do not want to render in the changelog
+  color: ff0000
```
Comment (zliang-akamai) [inline .github/labels.yml:24]: Maybe have a different color for `ignore-for-release`?

**Pull request merged**